website/static/js/profile.js

```diff
@@ -33,11 +33,25 @@ var cleanByRule = function(rule) {
 
 var noop = function() {};
 
+
+
+
+
 var SerializeMixin = function() {};
 
 /** Serialize to a JS Object. */
 SerializeMixin.prototype.serialize = function() {
-    return ko.toJS(this);
+    var serializedData = ko.toJS(this);
+    
+    // Remove blank websites from profileWebsites array before save
+    if (serializedData.urls.crud = "/api/v1/settings/social/") {
+        var profileWebsites = serializedData.trackedProperties[0];
```

Comment:
This seems brittle; what if we change the ordering of inputs? Or the order of trackedProperties?